src/JabbR-Core/Hubs/Chat.cs

```diff
@@ -737,12 +781,17 @@ void INotificationService.NudgeRoom(ChatRoom room, ChatUser user)
 
         void INotificationService.LeaveRoom(ChatUser user, ChatRoom room)
         {
+
             LeaveRoom(user, room);
         }
 
         private void LeaveRoom(ChatUser user, ChatRoom room)
         {
             var userViewModel = new UserViewModel(user);
+            
+            //TODO Remove explicit hub call
```

Comment:
doh - he was create room, you were leave. Why is the explicit hub call there though and wasn't previously?